net/device/IotHub.cs

```diff
@@ -261,6 +262,7 @@ private void ConnectionStatusChangesHandler(ConnectionStatus status, ConnectionS
                     {
                         { "DisconnectedStatus", _disconnectedStatus.ToString() },
                         { "DisconnectedReason", _disconnectedReason.ToString() },
+                        { "ConnectionStatusChangeCount", _connectionStatusChangeCount.ToString() },
```

Comment:
I log this because the order of traces is not guaranteed, and I want to know the order of dis/connection events as seen by the app.